.gitmodules

```diff
@@ -14,6 +14,7 @@
 [submodule "src/depends/Schnorr"]
 	path = src/depends/Schnorr
 	url = https://github.com/Zilliqa/schnorr.git
+	branch = fix_bn2
```

Comment:
this points to this branch https://github.com/Zilliqa/schnorr/pull/14 - which is needed for parsing the pubkey in uncompressed format (could work around to compressed format if need be but it is untidy)